examples/k8s/spire-agent.yaml

```diff
@@ -120,34 +110,40 @@ spec:
           args: ["-t", "30", "spire-server:8081"]
       containers:
         - name: spire-agent
-          image: gcr.io/spiffe-io/spire-agent:0.8.0
+          image: gcr.io/spiffe-io/spire-agent:0.7.3
```

Comment:
Was this version change/pin intentional? Seems like we should be using local images instead? This version doesn't include kubelet secure port which is configured above

Reply:
Oh yep. Missed this after resurrecting my old local branch. Totally unintentional. Will fix.